### DISCUSSION TITLE
Rendering Lottie

### DISCUSSION THREAD

**Discussion by dhayford92** [flet-dev/flet#83] [Ideas]
I think if we should have a flet package that will help parses Adobe After Effects animations exported as json with Bodymovin.

**Comment by FeodorFitsner** [comment]
This one: https://pub.dev/packages/lottie?

**Reply by dhayford92** [reply]
Yes please

**Comment by FeodorFitsner** [comment]
New issues created: https://github.com/flet-dev/flet/issues/88

**Comment by TingTingin** [comment]
what's the status of this?
